assets/src/application/ui/layout/entityList/filterBar/EntityListFilterBar.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { useState } from 'react';
 
-import CardViewFilterButton from './buttons/CardView';
+import CardViewFilterButton from './buttons/CardViewFilterButton';
 import Collapsible from './Collapsible';
 import { EntityListFilterBarProps } from './types';
 import { EntityListLegend } from '@application/ui/display';
-import TableViewFilterButton from './buttons/TableView';
-import ToggleLegendButton from './buttons/ToggleLegend';
-import ToggleFiltersButton from './buttons/ToggleFilters';
```

Comment:
When EDTR crashed with React Hook rules violation, it showed `CardView.tsx` and `TableView.tsx`, I was looking at EntityList card view components of dates and tickets 🙄